contributed_definitions/nyaml/NXmpes.yaml

```diff
@@ -135,107 +337,345 @@ NXmpes(NXobject):
         exists: optional
         doc: |
           Manipulator for positioning of the sample.
-        sample_temperature(NX_FLOAT):
+        temperature_sensor(NXsensor):
           exists: recommended
-          unit: NX_TEMPERATURE
-        drain_current(NX_FLOAT):
+          name:
+            exists: recommended
+          measurement:
+            enumeration: [temperature]
+          type:
+            exists: optional
+          value(NX_FLOAT):
+            exists: recommended
```

Comment:
Shouldn't we make this required if a sensor is provided? A sensor without a value does not make sense to me.